internal/audit/kafka/conf.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021-2023 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package kafka
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/cerbos/cerbos/internal/audit"
+)
+
+const confKey = audit.ConfKey + ".kafka"
+
+const (
+	EncodingJSON     = "json"
+	EncodingProtobuf = "protobuf"
+)
+
+// Conf is optional configuration for kafka Audit.
+type Conf struct {
+	// Seed brokers Kafka client will connect to
+	Brokers []string `yaml:"brokers" conf:",example=localhost:9092,localhost:9093"`
+	// Name of the topic audit entries are written to
+	Topic string `yaml:"topic" conf:",example=cerbos.audit.log"`
+	// Data format written to Kafka, accepts either json (default) or protobuf
+	Encoding string `yaml:"format" conf:",example=protobuf"`
+	// Improve performance by not waiting for confirmation from Kafka
+	Async bool `yaml:"async" conf:",example=true"`
```

Comment:
I'd rename this to `ProduceSync`. Then, if the config value is not set explicitly, we'll default to async -- which is better for performance..